plugins/exilepearl-paper/src/main/java/com/devotedmc/ExilePearl/storage/MySqlStorage.java

```diff
@@ -202,75 +212,152 @@ private boolean applyMigration0002() {
         return false;
     }
 
+    private boolean applyMigration0003() {
+        try (final Connection conn = this.db.getConnection()) {
+            try (final PreparedStatement stmt = conn.prepareStatement("ALTER TABLE exilepearls ADD CONSTRAINT unique_pearl_id UNIQUE (pearl_id);")) {
+                stmt.execute();
+            }
+            try (final PreparedStatement stmt = conn.prepareStatement(
+                """
+                create table if not exists ep_capturelocations(
```

Comment:
can we use the same table instead of creating a new table?

Reply:
Could do, I was only following the example of return locations, which are their own table. But if you'd prefer it as part of the `exilepearls` table, that's fine.